base/utils/R/get.analysis.filenames.r

```diff
@@ -22,42 +40,44 @@ ensemble.filename <- function(settings, prefix = "ensemble.samples", suffix = "R
     # run.
     ensemble.id <- "NOENSEMBLEID"
   }
-  
+
   ensemble.dir <- settings$outdir
-  
+
   dir.create(ensemble.dir, showWarnings = FALSE, recursive = TRUE)
-  
+
   if (all.var.yr) {
     # All variables and years will be included; omit those from filename
     ensemble.file <- file.path(ensemble.dir, paste(prefix, ensemble.id, suffix, sep = "."))
   } else {
-    ensemble.file <- file.path(ensemble.dir, paste(prefix, ensemble.id, variable, 
+    ensemble.file <- file.path(ensemble.dir, paste(prefix, ensemble.id, variable,
                                                    start.year, end.year, suffix, sep = "."))
   }
-  
+
   return(ensemble.file)
 } # ensemble.filename
 
 
 ##' Generate sensitivity analysis filenames
-##' 
+##'
 ##' @name sensitivity.filename
 ##' @title Generate sensitivity analysis filenames
-##' 
+##' @inheritParams ensemble.filename
+##' @param pft name of PFT used for analysis. If NULL, assumes all
+##'   PFTs in run are used and does not add them to the filename
 ##' @return a filename
 ##' @export
 ##'
 ##' @details  Generally uses values in settings, but can be overwritten for manual uses
 ##' @author Ryan Kelly
-sensitivity.filename <- function(settings, 
-                              prefix = "sensitivity.samples", suffix = "Rdata", 
+sensitivity.filename <- function(settings,
```

Comment:
maybe I'm misremembering, but weren't you moving the sensitivity functions to the uncertainty package?

Reply:
We are but we missed this file. @moki1202 I think this will be resolved when you merge https://github.com/moki1202/pecan/pull/5 (assuming you agree with my fix!)